### DISCUSSION TITLE
[Feature] Provide estimate on backup sizes

### DISCUSSION THREAD

**Discussion by KiruPoruno** [immich-app/immich#13969] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, you've to manually find out what the size of the singular database is, and then multiply that by the amount of backups your making, it would be useful to simply show an estimate for how big the backups will be at the current database size, inside "Backup Settings"

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Immich isn't aware of what backup tool you use etc, so could never give a meaningful estimate (as most good backup tools will do incremental backups & deduplication).
